cycledash/templates/comments.html

```diff
@@ -0,0 +1,13 @@
+{% extends "layouts/layout.html" %}
```

Comment:
I don't think it's too bad, since it's namespaced. I think of macro files as a collection, in theory, of related macros (parameterized templates). We could stick some more macros inside of here with other names related to rendering comments.